projects/kit/components/badged-content/badged-content.component.ts

```diff
@@ -22,11 +22,11 @@ const BADGE_SIZE: {[key: string]: TuiSizeL | TuiSizeXS} = {
 export class TuiBadgedContentComponent {
     @Input()
     @HostBinding('class._with-top')
-    contentTop: PolymorpheusContent;
+    contentTop: PolymorpheusPrimitive;
```

Comment:
It can be full `PolymorpheusContent` here. Probably just need to change `contentTop/Bottom` to `content` in the template.

Reply:
fixed